src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/AddressServiceImpl.java

```diff
@@ -1,15 +1,95 @@
 package uk.gov.ons.ctp.integration.contactcentresvc.service.impl;
 
-import com.godaddy.logging.Logger;
```

Comment:
GoDaddy logging is the RM vogue logger - some debug logging in this class wouldn't go amiss?

Reply:
Added logging.